tests/test_main.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from typer.testing import CliRunner
+
+from tollway.__main__ import app
+
+runner = CliRunner()
+
+
+@pytest.mark.parametrize(
+    "test_input, expected",
+    [
+        (["--total-events", 10, "--event-rate", 0.01], 0),
+        (["--total-events", "ten"], 2),  # invalid type
```

Comment:
You can also pytest.param() with id= instead of inline comments, eg:

<img width="1591" alt="image" src="https://github.com/BenGriffith/tollway-traffic/assets/387927/018e9368-a752-4f10-bfa7-53293ea14012">

Reply:
this is really cool